<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkPlot</class>
 <widget class="QDialog" name="NetworkPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="widget_plotnetuse_lo" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>860</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget_plotnetuse_eth" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>860</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget_plotnetuse_wl" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>860</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_plot_lo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Loopback</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_plot_eth">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ethernet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_plot_wl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Wireless</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
